iati_dashboard/templates/licenses.html

```diff
@@ -24,7 +24,7 @@ <h2 class="dashboard-panel-heading__title">Count of publishers per licences in u
                     <td>{% if license_urls[license]['url'] %}<a href="{{ license_urls[license]['url'] }}">{{ license_names[license] }}</a>{% else %}{{ license_names[license] }}{% endif %}</td>
                     <td>{{ license }}</td>
                     <td>{{ files }}</td>
-                    <td><a href="{{ url('dash-dataquality-licenses-detail', args=[license]) }}">{{ publisher_license_count[license] }}</a></td>
+                    <td><a href="{{ url('dash-licenses-detail', args=[license]) }}">{{ publisher_license_count[license] }}</a></td>
```

Comment:
dataquality hasn't been replaced with 'errors' here, as in the above replacements, but with nothing, whereas below dataquality has been put under the 'exploring-data' section?  Is this more generic licensing info somehow?